tests/std/tests/P1004R2_constexpr_vector_bool/test.cpp

```diff
@@ -0,0 +1,614 @@
+
+        const auto b = range_constructed.back();
+        static_assert(is_same_v<remove_const_t<decltype(b)>, _Iter_ref_t<vec::iterator>>);
+        assert(b);
+
+        const auto cb = const_range_constructed.back();
+        static_assert(is_same_v<remove_const_t<decltype(cb)>, _Iter_ref_t<vec::const_iterator>>);
+        assert(cb);
+#endif // !defined(__EDG__) || _ITERATOR_DEBUG_LEVEL != 2
+#endif // __EDG__
+    }
+
+    { // capacity
+#ifndef __EDG__ // TRANSITION, VSO-1274387
+        vec range_constructed(begin(input), end(input));
+
+        const auto e = range_constructed.empty();
+        static_assert(is_same_v<remove_const_t<decltype(e)>, bool>);
+        assert(e == false);
+
+        const auto s = range_constructed.size();
+        static_assert(is_same_v<remove_const_t<decltype(s)>, std::size_t>);
+        assert(s == size(input));
+
+        const auto ms = range_constructed.max_size();
+        static_assert(is_same_v<remove_const_t<decltype(ms)>, std::size_t>);
+        assert(ms == static_cast<std::size_t>(std::numeric_limits<ptrdiff_t>::max()));
+
+        range_constructed.reserve(20);
+
+        const auto c = range_constructed.capacity();
+        static_assert(is_same_v<remove_const_t<decltype(c)>, std::size_t>);
+        assert(c == 32);
+
+        range_constructed.shrink_to_fit();
+
+        const auto c2 = range_constructed.capacity();
+        static_assert(is_same_v<remove_const_t<decltype(c2)>, std::size_t>);
+        assert(c2 == 32);
+#endif // __EDG__
+    }
+
+    { // modifiers
+#ifndef __EDG__ // TRANSITION, VSO-1274387
+        vec range_constructed(begin(input), end(input));
+
+        vec cleared = range_constructed;
+        cleared.clear();
+        assert(cleared.empty());
+        assert(cleared.capacity() == range_constructed.capacity());
+
+        vec inserted;
+
+        const bool to_be_inserted = true;
+        inserted.insert(inserted.begin(), to_be_inserted);
+        assert(inserted.size() == 1);
+        assert(inserted.front());
+
+        const bool to_be_inserted2 = false;
+        inserted.insert(inserted.cbegin(), to_be_inserted2);
+        assert(inserted.size() == 2);
+        assert(inserted.front() == false);
+
+        inserted.insert(inserted.begin(), true);
+        assert(inserted.size() == 3);
+        assert(inserted.front());
+
+        inserted.insert(inserted.cbegin(), false);
+        assert(inserted.size() == 4);
+        assert(inserted.front() == false);
+
+#if !defined(__EDG__) || _ITERATOR_DEBUG_LEVEL != 2 // TRANSITION, VSO-1273381
+        const auto it = inserted.insert(inserted.begin(), begin(input), end(input));
+        assert(inserted.size() == 10);
+        assert(it == inserted.begin());
+
+        const auto it2 = inserted.insert(inserted.cbegin(), begin(input), end(input));
+        assert(inserted.size() == 16);
+        assert(it2 == inserted.begin());
+
+        const auto it3 = inserted.insert(inserted.begin(), {true, false, true});

```

Comment:
I think we may be missing `iterator insert(const_iterator position, size_type n, const bool& x)` overload test coverage?